smdebug/core/utils.py

```diff
@@ -49,18 +50,18 @@ class FRAMEWORK(Enum):
 _smddp_tf_imported = None
 _smddp_pt_imported = None
 _is_using_smmodelparallel = None
+_smp_imported = None
```

Comment:
where is this `_smp_imported` used in the codebase? I can't seem to find any other places that make use of the result.

I only see one place in `smdebug/core/config_validator.py` that uses the `check_smmodelparallel_training()` function.

Maybe, we should remove this part entirely?

Reply:
https://github.com/yl-to/sagemaker-debugger/blob/56fabe531692403e77ce9b5879d55211adec238e/smdebug/core/utils.py#L449
Please check the above line, here is the calling chain `_smp_imported`->`core/utils.py:get_distributed_worker` -> `core/utils.py:get_node_id`, and `get_node_id` is called in several places in core folder.